### PR TITLE
fix(COMP-2541): Improve performance on dynamic column

### PR DESCRIPTION
**Pull request by wwang-talend**
### Requirements

* Any code change adding any logic **MUST** be tested through a unit test executed with the default build
* Any API addition **MUST** be done with a documentation update if relevant 

### Why this PR is needed?
https://jira.talendforge.org/browse/TCOMP-2541

### What does this PR adds (design/code thoughts)?
https://jira.talendforge.org/browse/TCOMP-2541

### CODE REVIEW

File: component-studio/component-runtime-di/src/main/java/org/talend/sdk/component/runtime/di/record/DiRecordVisitor.java [hunk 1]
```diff
@@ -31,16 +32,7 @@
...
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.OptionalDouble;
-import java.util.OptionalInt;
-import java.util.OptionalLong;
-import java.util.Set;
+import java.util.*;
```
Comment (undx) [inline component-studio/component-runtime-di/src/main/java/org/talend/sdk/component/runtime/di/record/DiRecordVisitor.java:35]: do not use star import please.

**Review by undx — COMMENTED** [review]
just one comment

File: component-studio/component-runtime-di/src/main/java/org/talend/sdk/component/runtime/di/record/DiRecordVisitor.java (hunk 1 above)
Comment (undx) [inline component-studio/component-runtime-di/src/main/java/org/talend/sdk/component/runtime/di/record/DiRecordVisitor.java:35]: do not use star import please.

**Review by undx — COMMENTED** [review]
just one comment. Otherwise LGTM

File: component-studio/component-runtime-di/src/main/java/org/talend/sdk/component/runtime/di/record/DiRecordVisitor.java (hunk 1 above)
Comment (wwang-talend) [reply component-studio/component-runtime-di/src/main/java/org/talend/sdk/component/runtime/di/record/DiRecordVisitor.java]: thanks a lot, just fixed.

**Review by undx — APPROVED** [review]
LGTM

**Pull request merged**